src/reducers/userReducer.js

```diff
@@ -0,0 +1,53 @@
+
+export default function reducer(state = {
```

Comment:
Don't use default exports. Here's why: https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad

Reply:
create an `INITIAL_STATE` object above the function and use it as default value for state.